mods/ts/mod.yaml

```diff
@@ -396,3 +396,56 @@ ModContent:
 					^Content/ts/temperat.mix:
 						Offset: 73056940
 						Length: 2037606
+		origin: C&C The Ultimate Collection (Origin version)
+			Type: Install
+			RegistryKey: HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\EA Games\Command and Conquer Tiberian Sun
+			RegistryValue: Install Dir
+			IDFiles:
+				TSLauncher.exe: 0b3e8dfd7cb701868eccef3ba3717a1ced802707
+				GDFBinary_en_US.dll: 4bb56a449bd0003e7ae67625d90a11ae169319d6
+			Install:
+				copy: .
+					^Content/ts/scores.mix: SCORES.MIX
+				extract-raw: TIBSUN.MIX
```

Comment:
You can make this work with the localized versions as well if you don't rely on the file sizes of the .mix files so it might be smart to just copy the TIBSUN.MIX here.